perf_dashboard/benchmarks/views.py

```diff
@@ -25,6 +25,21 @@
 cpu_cur_selected_release = []
 cpu_master_selected_release = []
 current_release = [os.getenv('CUR_RELEASE')]
+release_days = ['30', '60', '90']
+cur_selected_days = []
```

Comment:
Is the state of `cur_selected_days` per user session or global (i.e. if one user selects `60` will another user see `60` by default or when they refresh)?

Reply:
I did not introduce user concept in this app right now. Might not be necessary? or later on we can add it? Therefore, this is globally thing. Since I thought if there is one select like "90" then, we have downloaded that 90 release files, we can keep it there to display on the UI?

Reply:
let's keep all things simple for now, the main thing is to realize those main features, and later on we can do any kind of refactor works or so. That's my thought 🤔

Reply:
The transient state of inputs in UI should not be globally shared state. This means one users selection can affect all other users.... But that is outside the scope of this change, so can be something for future work.